### PR TITLE
[Mellanox] added component versions to techsupport

### PR DESCRIPTION
**Pull request by skr31**
#### What I did
added component versions to techsupport

### CODE REVIEW

File: scripts/generate_dump [hunk 1]
```diff
@@ -1251,6 +1251,7 @@ collect_mellanox() {
       echo "HW Mgmt dump script $HW_DUMP_FILE does not exist"
     fi
 
+    save_cmd "get_component_versions.py" "component_versions"
```
Comment (dprital) [inline scripts/generate_dump:1254]: Are you sure this is the file to keep ? This is the script file, we need the file that contains all the packages

**Comment by dprital** [comment]
@skr31 , I belive that this PR must arrive together with the PR on sonic-buildimge.
What will happen if this this change will be adopted while the PR on sonic-buildimage will not ? Will it break SONiC ?

File: scripts/generate_dump (hunk 1 above)
Comment (skr31) [reply scripts/generate_dump:1254]: The output of the script file is saved into component_versions

**Review by dprital — APPROVED** [review]

File: scripts/generate_dump (hunk 1 above)
Comment (nazariig) [inline scripts/generate_dump:1254]: @skr31 what if docker is not running? Please handle accordingly.

**Review by nazariig — CHANGES_REQUESTED** [review]

File: scripts/generate_dump (hunk 1 above)
Comment (skr31) [reply scripts/generate_dump:1254]: If syncd is not running, the relevant versions will be marked 'N/A'

Comment (nazariig) [reply scripts/generate_dump:1254]: @skr31 ok

**Review by nazariig — APPROVED** [review]

**Pull request closed without merging**